shell/src/test/java/alluxio/cli/LogLevelTest.java

```diff
@@ -107,19 +105,19 @@ public void parseZooKeeperHAMasterTarget() throws Exception {
     when(mockCommandLine.getArgs()).thenReturn(mockArgs);
     when(mockCommandLine.hasOption(LogLevel.TARGET_OPTION_NAME)).thenReturn(true);
     when(mockCommandLine.getOptionValue(LogLevel.TARGET_OPTION_NAME)).thenReturn(mockArgs[1]);
-
-    PowerMockito.mockStatic(MasterInquireClient.Factory.class);
+    MockedStatic theMock = Mockito.mockStatic(MasterInquireClient.Factory.class);
```

Comment:
1. You can statically import `mockStatic` now since we will be only using Mockito. No need to qualify that as `Mockito.mockStatic`.
2. Any particular reason to prefer manually closing a mock over a `try-with-resources` block here?
3. IDE is giving me a warning about raw generic use of `MockedStatic`. Try `MockedStatic<MasterInquireClient.Factory>`.